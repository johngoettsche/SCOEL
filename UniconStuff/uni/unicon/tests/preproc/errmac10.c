/* test error message: illegal redefinition of __DATE__ */

#begdef __DATE__
12/03/01
#enddef
