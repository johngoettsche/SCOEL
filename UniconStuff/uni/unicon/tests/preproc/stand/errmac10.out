pp: File errmac10.c; Line 3: invalid redefinition of macro __DATE__
